trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -246,11 +257,31 @@ subroutine read_rt_nlst(nlst)
   nlst%DEEPGWSPIN = DEEPGWSPIN
   nlst%SOLVEG_INITSWC = SOLVEG_INITSWC
   nlst%reservoir_obs_dir = "testDirectory"
+  nlst%reservoir_persistence = reservoir_persistence
+  nlst%reservoir_persistence_parameter_file = reservoir_persistence_parameter_file
+  nlst%reservoir_timeslice_path = reservoir_timeslice_path
+  nlst%reservoir_observation_lookback_hours = reservoir_observation_lookback_hours
+
+  ! If in retrospective mode, set default reservoir_observation_update_time_interval_seconds to one day
+  if ( io_config_outputs .eq. 1 .or. io_config_outputs .eq. 5 ) then
```

Comment:
io_config_outputs is not a good way to detect "forecast/non-forecast" - as per a previous discussion. 
I said I would make changes to the hydro.namelist to add a forecast inidcator switch... I should do that with this PR>

Reply:
Regarding `reservoir_observation_update_time_interval_seconds`
1000000s/(60*60*24s)/day < 12 days
is the the typical persistence time much less than 12 days? In a 30-day forecast, if the module finds an observations 12 days ago, will it have an effect?

Could a negative number be used here to short circuit looking for observations entirely? Or, better yet, can that same short circuit be achieved by the "forecast" flag that will be added. Then this value will not need changed.

Reply:
I agree. I kept io_config_outputs for the time being, so that the sims would work the way we wanted.

We didn't want any observation ever picked up beyond the first timestep with any length of forecast, so we set it to 1 billion seconds. We max out our persistence at 10 days but have it mostly attenuated the last few days and give most weight to level pool.

We have a gage lookback time, which would ideally be set to about 18 or 24 hours. This would prevent grabbing an observation looking back beyond that amount of time before the start time of the forecast. For the 30-day forecast, all weight would be given to level pool after 10 days. The update interval is used in AnA to read obs at an example interval of every 3 days and persist that value. 

A negative number could not be used because our code reads the obs at the first timestep and then sets the update time for the next obs read. A negative number would cause it to continuously read new obs because future timesteps will see that it is past the update time. 

https://github.com/jdmattern-noaa/wrf_hydro_nwm_public/blob/persistence/trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid.F#L255

We thought about using the forecast flag in that code also, but I think decided that it would add more complexity than was needed.  You bring up a good point, and we could revisit this though if you want.

Reply:
@jmccreight Will add the `forecast` field to the hydro.namelist. Then we can work on fixing this.